Changes

```diff
@@ -71,6 +71,14 @@ Working version
    (Whitequark and Jacques-Henri Jourdan, review by Gabriel Scherer
     and Xavier Clerc)
 
+* #8833, `ocaml`: adhere to the XDG base directory specification to
+  lookup an `.ocamlinit` file. Reads an `$XDG_CONFIG_HOME/ocaml/init.ml`
+  file before trying to lookup `~/.ocamlinit`. On Windows: `%LOCALAPPDATA%`
+  or %APPDATA% is also consulted if the `XDG_CONFIG_HOME` variable is
+  undefined.
+  (Daniel C. Bünzli, review by David Allsopp, Armaël Guéneau and
+   Nicolás Ojeda Bär)
+
```

Comment:
GitHub suggestions is a bit naff for multiple lines:

```diff
diff --git a/Changes b/Changes
index e789c378d2..dd4eaec3a5 100644
--- a/Changes
+++ b/Changes
@@ -71,11 +71,10 @@ Working version
    (Whitequark and Jacques-Henri Jourdan, review by Gabriel Scherer
     and Xavier Clerc)
 
-* #8833, `ocaml`: adhere to the XDG base directory specification to
-  lookup an `.ocamlinit` file. Reads an `$XDG_CONFIG_HOME/ocaml/init.ml`
-  file before trying to lookup `~/.ocamlinit`. On Windows: `%LOCALAPPDATA%`
-  or %APPDATA% is also consulted if the `XDG_CONFIG_HOME` variable is
-  undefined.
+* #8834, `ocaml`: adhere to the XDG base directory specification to locate an
+  `.ocamlinit` file. Reads `$XDG_CONFIG_HOME/ocaml/init.ml` before trying
+  `~/.ocamlinit`. On MSVC/mingw-w64 builds, `%LOCALAPPDATA%` and `%APPDATA%`
+  are also consulted if the `XDG_CONFIG_HOME` variable is undefined.
   (Daniel C. Bünzli, review by David Allsopp, Armaël Guéneau and
    Nicolás Ojeda Bär)
```
(the important ones are the wrong PR number and the missing backticks on `` `%APPDATA%` `` - the rest are suggestions)